api/custom_auth/urls.py

```diff
@@ -26,7 +26,11 @@
         CustomAuthTokenLoginWithMFACode.as_view(),
         name="mfa-authtoken-login-code",
     ),
-    path("logout/", TokenDestroyView.as_view(), name="authtoken-logout"),
+    path(
+        "logout/",
+        TokenDestroyJWTSlidingBlacklistView.as_view(),
+        name="authtoken-logout",
+    ),
```

Comment:
Hmm... shouldn't we only add this if the setting is enabled?

Reply:
Good point, done.

Reply:
Reversed this to simplify testing. The endpoint is a no-op for classic token auth anyway now.